code/modules/tooltip/tooltip.dm

```diff
@@ -81,15 +81,16 @@ Notes:
 
 /datum/tooltip/proc/hide()
 	if (src.queueHide)
-		spawn(1)
-			winshow(src.owner, src.control, 0)
+		addtimer(CALLBACK(src, .proc/do_hide), 1)
```

Comment:
or wait what I didn't write this I didn't realize what you mean by tooltips this is wirewraith(?)'s tooltip code.